checkstyle/import-control.xml

```diff
@@ -63,6 +63,8 @@
       <allow pkg="org.apache.kafka.common.config" />
       <allow pkg="org.apache.kafka.common.metrics" />
       <allow pkg="org.apache.kafka.common.security" />
+      <!-- for testing -->
+      <allow pkg="org.apache.kafka.common.requests" />
```

Comment:
@ijuma Yes, I didn't like doing this either. I have moved `SaslAuthenticatorTest` to `o.a.k.c.security.authenticator` instead of `o.a.k.c.network`. The test uses the test server/util classes from `o.a.k.c.network` (which was the reason I had the test there in the first place), but perhaps this is a better organization? Let me know what you think.